ts/project.ts

```diff
@@ -22,11 +22,45 @@ export type ProjectData = {
   defaultExecutable?: Executable;
 };
 
+/**
+ * `Plugin`s automatically add one or more fields to your project. You can use
+ * existing plugins to automatically add a few useful fields without having to
```

Comment:
```suggestion
 * existing plugins to automatically add prepackaged functionality to your `Project`.
```

(I think talk of "fields", which is quite structural, should be combined with talk of *function*.)